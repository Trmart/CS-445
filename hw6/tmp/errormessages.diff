4a5
> ERROR(17): '+' requires operands of type int but lhs is of type void.
9c10
< Number of errors: 8
---
> Number of errors: 9
