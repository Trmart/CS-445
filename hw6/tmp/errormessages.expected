ERROR(10): Initializer for variable 'c' of type char is of type int
ERROR(13): Expecting Boolean test condition in while statement but got type char.
ERROR(14): Unary 'chsign' requires an operand of type int but was given type char.
ERROR(15): Unary '--' requires an operand of type int but was given type char.
ERROR(17): Symbol 'emu' is not declared.
ERROR(17): Symbol 'fox' is not declared.
ERROR(5): Function 'cat' at line 5 is expecting to return type char but returns type int.
ERROR(7): Function 'dog' at line 7 is expecting to return type int but return has no value.
Number of errors: 8
Number of warnings: 4
WARNING(11): The variable 'i' seems not to be used.
WARNING(5): The function 'cat' seems not to be used.
WARNING(7): The function 'dog' seems not to be used.
WARNING(9): Expecting to return type int but function 'main' has no return statement.
